docs/getting-started/provider-playbook/technology-providers-support/sdks-and-development-apis.md

```diff
@@ -0,0 +1,36 @@
+# SDKs and Development APIs
+
+:::info Key Concepts
+
+* Ed-Fi's technology efforts use Swagger/OpenAPI code generation to allow developers to generate SDKs for their platform of choice.
+* If you are working on an API client for an API that has been extended, you will need to get the Swagger/OpenAPI definition from the API host.
+
+:::
+
+## Software Development Kits
+
+Software Development Kits (SDKs) can be very useful for developers of API clients.  For those technology providers connecting to an API implemented by the Ed-Fi ODS API, there are instructions for generating an SDK.
+
+* [Using Code Generation to Create an SDK](https://edfi.atlassian.net/wiki/spaces/ODSAPI31/pages/26968911/Using+Code+Generation+to+Create+an+SDK) (latest guidance for Suite 3)
+
+The Ed-Fi ODS API generation process uses OpenAPI (or OpenAPI's predecessor, Swagger) specification files to generate the SDK.
+
+Accordingly if the API implementer has extended the API you are connecting to, you will need to get the OpenAPI specification from the API host in order to generate an SDK that has the extensions included.
+
+## Cloud Deployments of the ODS API
+
+In developing, it can be very helpful to have an actual API to work against. In the section, [Getting Started with Ed-Fi APIs](../technology-providers-implementation/getting-started-with-apis/readme.md), you were introduced to the Ed-Fi API Sandbox, which is a public service for experimenting with Ed-Fi APIs.
+
+However, it is **not** appropriate to introduce the Ed-Fi API Sandbox into your development or QA processes. To assist you in those processes, there are some cloud-based deployments that simplify the process of installing a cloud-based version of the Ed-Fi ODS API, which you can then use for your internal development work.
```

Comment:
Why? I don't understand this point.

Reply:
I believe it's to avoid building automation around an instance where updates are not controlled by the implementor. My understanding is that the Ed-Fi instance gets reset fairly often so you wouldn't want to use it cases where you expect a previous state to be maintained.

Reply:
OK, even though it says on line 22 "public service..." I wasn't picking up that this reference to "Ed-Fi API Sandbox" means https://api.ed-fi.org. I don't think I've heard someone call that the "Ed-Fi API Sandbox" before. At some point while reviewing the documentation yesterday I realized this disconnect on my part, but didn't come back to this comment after realizing that.

To me, it is just "api.ed-fi.org" or the "documentation site" or "public demonstration site". "Ed-Fi API Sandbox" to mean implied running the ODS/API in sandbox mode.

Let's leave the language as is, because it was probably clear to someone not deep in the tech side of things. However, we should revisit in the future and also point out that they install on their own systems (localhost or on prem), not just cloud, using the installation instructions we provide and/or Docker deployment.

Reply:
Looking at where it was referenced in the getting started, it says:

**Ed-Fi ODS / API sandbox** at [https://api.ed-fi.org/](https://api.ed-fi.org/v7.2/docs/swagger/index.html?urls.primaryName=Resources)

I'm going to update it here to be the same.